quarkus/tests/integration/src/test/java/org/keycloak/it/storage/database/dist/DatabaseOptionsDistTest.java

```diff
@@ -35,4 +36,12 @@ void testSetSchema(LaunchResult result) {
         CLIResult cliResult = (CLIResult) result;
         cliResult.assertStartedDevMode();
     }
+
+    @Test
+    void testEnvVarPrecedenceOverConfFile(KeycloakDistribution distribution) {
```

Comment:
Quarkus tests are failing atm, most likely because the Env Var is still set and taken then also for other tests. Please add a method that clears it :)